docker-compose-configuration/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "2.2"
```

Comment:
is there a reason we use 2.2?

Reply:
yes, later versions do not support network linking in the same way.